### ISSUE TITLE
Support other provinces, countries

### ISSUE THREAD

**Issue by millette** [millette/got-weed#4]
Ontario, etc.

**Comment by samuelwang48** [comment]
Hmmm interesting... This is getting ambitious:) 

2 cents
1. Need to do some analysis on all the official provincial website first. This could be a markdown file.
2. A transformation layer (function) is needed to deal with different data sources when we design.

**Comment by millette** [comment]
Yes, I don't even know what other places there are. Let's use the [wiki](https://github.com/millette/got-weed/wiki/Online-stores) for that.

No need to do everything at once either. But I'd really like to compare things across countries/provinces on a public website #7, eventually.
